### PR TITLE
Feat: added card component

### PR DESCRIPTION
**Pull request by ArjunGTX**
Added Card component. 
ezgif-2-6e91c1f5fe

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
nice work

File: css/components/card.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+.card {
+    width: 11rem;
+    height: max-content;
+    display: flex;
+    flex-direction: column;
+    border-radius: 0.625rem;
+    overflow: hidden;
+    box-shadow: var(--shadow-white);
+    background-color: var();
```
Comment (devananth) [inline css/components/card.css]: You can remove this line

**Review by devananth — COMMENTED** [review]
Looks good !

File: css/components/card.css (hunk 1 above)
Comment (ArjunGTX) [reply css/components/card.css]: Ok . Thankyou for pointing it out

Comment (ArjunGTX) [reply css/components/card.css]: I have fixed this part

**Review by SukanyaSen530 — APPROVED** [review]
Card Design looks premium :) LGTM

**Pull request merged**